docs/pbc/all/request-for-quote/use-cases-request-for-quote-processes.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Use cases: Request for Quote processes"
+description: This document describes the use cases for Requests for Quote.
```

Comment:
Here I'd say its either Requests for Quotes or Request for Quote, the first if its a generic term, the second if its a name.

Reply:
Request for Quote is the name of the feature.

Reply:
description: This document describes the use cases for Request for Quote.

then I think would be best